soap/settings.py

```
@@ -0,0 +1,35 @@
+"""
+تنظیمات اختصاصی اپ SOAP
+
+توجه: مطابق دستور کار، تنظیمات در اپ قرار می‌گیرد و نیازی به تغییر
+آدرس‌های مرجع ریشه پروژه نیست.
+"""
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)